files/class-api-client.php

```diff
@@ -122,6 +131,9 @@ public function upload_file( $local_path, $upload_path ) {
 		}
 
 		// response looks like {"filename":"/wp-content/uploads/path/to/file.ext"}
+		// save to cache
+		$this->cache->cache_file( $response_data->filename, file_get_contents( $local_path ) );
```

Comment:
We'll want to clear out the cache for a successful `delete_file` call as well.

Reply:
Good point! I'll add that in.